src/ResourceManager/Sql/Commands.Sql/Database/Model/AzureSqlDatabaseModel.cs

```diff
@@ -83,21 +84,11 @@ public class AzureSqlDatabaseModel
         /// </summary>
         public DateTime CreationDate { get; set; }
 
-        /// <summary>
-        /// Gets or sets the current service objective ID
-        /// </summary>
-        public Guid CurrentServiceObjectiveId { get; set; }
```

Comment:
Removing these is a breaking change

Reply:
@jaredmoo 
1. -Edition, -RequestedServiceObjectName. Yes, I agree. After I tested the cmdlet, found that the returned database info on edition and serviceObjectiveName for vcore database will be: Edition: GeneralPurpose, CurrentServiceObjectiveName: GP_Gen4_2.

Can we use existing RequestServiceObjectiveName parameter to specify GP_Gen4_2? (including vcore number)

2. I removed the CurrentServiceObjectiveId and related reference on it. Since we're not using ServiceObjectiveId anymore in the Management.Sql database model. If keep it, the returned information for created/updated database will be empty '00000000-0000-0000-0000-000000000000'

Reply:
For my above reply. I think I should use existing -RequestServiceObjectiveName parameter to specify GP_Gen4, and add new parameter -Vcore to specify number of vcores